skypack/src/index.ts

```diff
@@ -196,7 +196,10 @@ export async function lookupBySpecifier(
     (semverString ? `@${semverString}` : ``) +
     (packagePath ? `/${packagePath}` : ``);
   try {
-    const {body, headers, isCached, isStale} = await fetchCDN(lookupUrl, userAgent);
+    const {body, statusCode, headers, isCached, isStale} = await fetchCDN(lookupUrl, userAgent);
+    if (statusCode !== 200) {
```

Comment:
Is there any other situation where we‘d return another `2xx` code? (or `3xx`?)

Reply:
Great call, we do only return 200s in this case, but this is one place where the `?meta` param will really help